src/main/php/webservices/rest/RestJsonSerializer.class.php

```diff
@@ -36,30 +39,32 @@ public function contentType() {
    */
   public function serialize($payload, $out) {
     $val= $payload instanceof Payload ? $payload->value : $payload;
+    $json= new StreamOutput($out, $this->format);
     if ($val instanceof \Traversable) {
       $i= 0;
```

Comment:
This code can be simplified once https://github.com/xp-forge/json/pull/7 is released